packages/geo/src/lib/draw/draw/draw.component.ts

```diff
@@ -0,0 +1,484 @@
+        private styleService: StyleService,
+        private formBuilder: FormBuilder,
+        private drawStyleService: DrawStyleService,
+        private dialog: MatDialog,
+    ) {
+        this.buildForm();
+        this.fillColor = this.drawStyleService.getFill();
+        this.strokeColor = this.drawStyleService.getStroke();
+        this.showTooltips = this.drawStyleService.getToggleLabel();
+        this.toggleLabel = this.drawStyleService.getToggleLabel();
+    }
+
+    ngOnInit() {
+        this.initStore();
+        this.createDrawPointControl();
+        this.createDrawLineControl();
+        this.createDrawPolygonControl();
+        this.createDrawCircleControl();
+        this.toggleDrawControl();
+        this.checkStoreCount();
+    }
+
+    /**
+     * Clear the overlay layer and any interaction added by this component.
+     * @internal
+     */
+    ngOnDestroy() {
+        this.setActiveDrawType(undefined);
+    }
+
+    createDrawPointControl(fill?: string, stroke?: string) {
+        this.drawPointControl = new DrawControl({
+            geometryType: DrawType.Point,
+            source: this.olDrawSource,
+            layerStyle: new OlStyle({}),
+            drawStyle: createDrawingInteractionStyle(this.fillColor, this.strokeColor)
+        });
+    }
+
+    createDrawLineControl(fill?: string, stroke?: string) {
+        this.drawLineControl = new DrawControl({
+            geometryType: DrawType.LineString,
+            source: this.olDrawSource,
+            layerStyle: new OlStyle({}),
+            drawStyle: createDrawingInteractionStyle(this.fillColor, this.strokeColor)
+        });
+    }
+
+    createDrawPolygonControl(fill?: string, stroke?: string) {
+        this.drawPolygonControl = new DrawControl({
+            geometryType: DrawType.Polygon,
+            source: this.olDrawSource,
+            layerStyle: new OlStyle({}),
+            drawStyle: createDrawingInteractionStyle(this.fillColor, this.strokeColor)
+        });
+    }
+
+    createDrawCircleControl(fill?: string, stroke?: string) {
+        this.drawCircleControl = new DrawControl({
+            geometryType: DrawType.Circle,
+            source: this.olDrawSource,
+            layerStyle: new OlStyle({}),
+            drawStyle: createDrawingInteractionStyle(this.fillColor, this.strokeColor)
+        });
+    }
+
+    onDrawTypeChange(drawType: DrawType) {
+        this.activeDrawType = drawType;
+    }
+
+    private initStore() {
+        const store = this.store;
+        this.map.removeLayer(this.layer);
+
+        this.layer = new VectorLayer({
+          title: this.languageService.translate.instant('igo.geo.draw.drawing'),
+          zIndex: 200,
+          source: new FeatureDataSource(),
+          style: (feature, resolution) => {
+            return this.drawStyleService.createDrawLayerStyle(feature, resolution, this.map, true);
+          },
+          showInLayerList: true,
+          exportable: false,
```

Comment:
You should let the layer being exportable. @drekss

Reply:
But there is an issue with the shapefile output...

Reply:
{"error":true,"msg":"ERROR 1: Attempt to write non-point (POLYGON) geometry to point shapefile.\nERROR 1: Unable to write feature 1 from layer ogr_57d2b442232.\nERROR 1: Terminating translation prematurely after failed\ntranslation of layer ogr_57d2b442232 (use -skipfailures to skip errors)\n"}

Reply:
Maybe split the features on export  by geometry type..
I.E.
if shapefile and multigeometryDraw:
    split by geomety and make a call to ogre for each geometry type.